src/Cake.SqlServer/SqlServerAliasesImpl.cs

```diff
@@ -164,6 +165,16 @@ internal static SqlConnection OpenSqlConnection(ICakeContext context, String con
                     var newException = new Exception(errorMessage, exception);
                     throw newException;
                 }
+#else
```

Comment:
I don't see a difference with the block of code above. Is `#if NET451` required here?